rskj-core/src/main/java/co/rsk/peg/BridgeSerializationUtils.java

```diff
@@ -61,19 +61,51 @@ private BridgeSerializationUtils() {
         throw new IllegalAccessError("Utility class, do not instantiate it");
     }
 
-    public static byte[] serializeRskTxWaitingForSignatures(
-          Map.Entry<Keccak256, BtcTransaction> rskTxWaitingForSignaturesEntry) {
-        byte[][] serializedRskTxWaitingForSignaturesEntry =
-            serializeRskTxWaitingForSignaturesEntry(rskTxWaitingForSignaturesEntry);
+    public static byte[] serializeRskTxHash(Keccak256 rskTxHash) {
+        return RLP.encodeElement(rskTxHash.getBytes());
+    }
+
+    private static Keccak256 deserializeRskTxHashFromRLPData(byte[] rlpData) {
+        return new Keccak256(rlpData);
+    }
+
+    public static byte[] serializeBtcTransaction(BtcTransaction btcTransaction) {
+        return RLP.encodeElement(btcTransaction.bitcoinSerialize());
+    }
+
+    public static BtcTransaction deserializeBtcTransaction(byte[] data, NetworkParameters networkParameters, boolean txHasInputs) {
+        if (data == null || data.length == 0) {
+            return null;
+        }
+
+        byte[] rlpData = RLP.decode2(data).get(0).getRLPData();
+        return deserializeBtcTransactionFromRLPData(rlpData, networkParameters, txHasInputs);
+    }
+
+    private static BtcTransaction deserializeBtcTransactionFromRLPData(byte[] rlpData, NetworkParameters networkParameters, boolean txHasInputs) {
+        if (!txHasInputs) {
+            BtcTransaction tx = new BtcTransaction(networkParameters);
+            tx.parseNoInputs(rlpData);
+            return tx;
+        }
+
+        return new BtcTransaction(networkParameters, rlpData);
+    }
+
+    public static BtcTransaction deserializeSvpFundTx(byte[] data, NetworkParameters networkParameters) {
+        return deserializeBtcTransaction(data, networkParameters, true);
+    }
+
+    public static byte[] serializeRskTxWaitingForSignatures(Map.Entry<Keccak256, BtcTransaction> rskTxWaitingForSignaturesEntry) {
+        byte[][] serializedRskTxWaitingForSignaturesEntry = serializeRskTxWaitingForSignaturesEntry(rskTxWaitingForSignaturesEntry);
         return RLP.encodeList(serializedRskTxWaitingForSignaturesEntry);
     }
 
-    public static byte[] serializeRskTxsWaitingForSignatures(
-          SortedMap<Keccak256, BtcTransaction> rskTxWaitingForSignaturesMap) {
+    public static byte[] serializeRskTxsWaitingForSignatures(SortedMap<Keccak256, BtcTransaction> rskTxWaitingForSignaturesMap) {
```

Comment:
Google Java Style Guide enforces a 100-character line limit, is there any reason why this is being changed across the class?

Reply:
my bad, sorry

Reply:
fixed